radixdlt/src/integration/java/com/radixdlt/consensus/checks/LivenessCheck.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.consensus.checks;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.radixdlt.consensus.BFTCheck;
+import com.radixdlt.consensus.BFTTestNetwork;
+import com.radixdlt.consensus.QuorumCertificate;
+import com.radixdlt.consensus.VertexStore;
+import com.radixdlt.consensus.View;
+import io.reactivex.rxjava3.core.Observable;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+import org.assertj.core.api.Condition;
+
+/**
+ * Check that the network is making progress by ensuring that QCs are progressively
+ * increasing.
+ */
+public class LivenessCheck implements BFTCheck {
+
+	@Override
+	public Observable<Object> check(BFTTestNetwork network) {
+		// there should be a new highest QC every once in a while to ensure progress
+		// the minimum latency per round is determined using the network latency
+		// a round can consist of 6 * MTT
```

Comment:
MTT meaning?

Reply:
max transmission time

Reply:
Please add a comment stating that, wasn't obvious to me